Core/HLE/proAdhoc.cpp

```diff
@@ -200,6 +203,15 @@ SceNetAdhocctlPeerInfo * findFriend(SceNetEtherAddr * MAC) {
 	return peer;
 }
 
+int getNonBlockingFlag(int fd) {
+#ifdef _WIN32
```

Comment:
Hm, you could use `ioctlsocket`.  What is this used for?

-[Unknown]

Reply:
We are currently switching the socket blocking & non-blocking.

From what i read at stackoverflow, on Windows there is no way to get the blocking state of a socket, so this getNonBlockingFlag is kinda broken on windows, and i guess it won't be needed anymore since this PR is simulating the blocking process so we can assume it will always be in non-blocking mode.